src/engraving/types/typesconv.cpp

```diff
@@ -231,6 +232,7 @@ static const std::vector<Item<ElementType> > ELEMENT_TYPES = {
     { ElementType::WHAMMY_BAR,           "WhammyBar",            TranslatableString("engraving", "Whammy bar") },
     { ElementType::RASGUEADO,            "Rasgueado",            TranslatableString("engraving", "Rasgueado") },
     { ElementType::HARMONIC_MARK,        "HarmonicMark",         TranslatableString("engraving", "Harmonic mark") },
+    { ElementType::PICK_SCRAPE,          "PickScrape",           TranslatableString("engraving", "Pick scrape out") },
```

Comment:
@alexpavlov96 Why is it called "Pick scrape _out_" here instead of just "Pick scrape"? Is that a typo?